code/modules/world_api/commands/server_query.dm

```diff
@@ -0,0 +1,295 @@
+//Get Server Status
+/datum/topic_command/get_serverstatus
+	name = "get_serverstatus"
+	description = "Gets the server status."
+	no_auth = TRUE
+
+/datum/topic_command/get_serverstatus/run_command(queryparams)
+	var/list/s[] = list()
+	s["version"] = game_version
+	s["mode"] = master_mode
+	s["respawn"] = config.abandon_allowed
+	s["enter"] = config.enter_allowed
+	s["vote"] = config.allow_vote_mode
+	s["ai"] = config.allow_ai
+	s["host"] = host ? host : null
+	s["stationtime"] = worldtime2text()
+	s["roundduration"] = get_round_duration_formatted()
+	s["gameid"] = game_id
```

Comment:
Maybe as we are refactoring could be have a way to alter data depending if user is authed or not, like game ID is somewhat sensitive, no? Then we could add if that checks for Auth and not send game ID to non authed requests. This thing could be used to protect other somewhat sensitive data, like player lists, while maintaining compatibility.

Reply:
None of this information is sensitive. And in theory, you can already add the sensitive data check by using a proc to check if the query is authed.

However, since end points should roughly do one thing, it might be a bad idea to split the information in two without setting up another endpoint.